smoketest/calendarsubscription_test.go

```diff
@@ -0,0 +1,69 @@
+package smoketest
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/target/goalert/smoketest/harness"
+)
+
+func TestICal(t *testing.T) {
```

Comment:
Follow the filename:
```suggestion
func TestCalendarSubscription(t *testing.T) {
```